### DISCUSSION TITLE
Bug: The app can't find a current location unless there is a mobile netowork signal available

### DISCUSSION THREAD

**Discussion by twisteros** [osmandapp/OsmAnd#13754] [Q&A]

**Comment by barteis** [comment]
Just OsmAnd?  Or do other GPS apps (e.g. Google Maps) do the same?

**Reply by vshcherb** [reply]
OsmAnd nightly build now have switch to use Google Service Location or pure Android location that might help

**Reply by twisteros** [reply]
> Just OsmAnd? Or do other GPS apps (e.g. Google Maps) do the same?

Other apps like WhatsApp, Waze, HereWeGo are finding location regardless of the mobile signal status. 

> OsmAnd nightly build now have switch to use Google Service Location or pure Android location that might help

Is it available in the latest F-Droid release?
